README.md

```diff
@@ -149,6 +149,24 @@ Example request body:
 }
 ```
 
+### **/cloudflare_captcha_solver**
```

Comment:
Better to merge with old endpoint for solving capthca. Maybe with some new params and defaults which leads to old solving.